### PR TITLE
GEOPY-967: fix error copy_from_extent on drillhole groups

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-967 - Error copy_from_extent on drillhole groups**

### CODE REVIEW

File: tests/geo_image_test.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ def test_create_copy_geoimage(tmp_path):
         np.c_[10.0, 10.0, 3],
     ]
 
-    geoimage = GeoImage.create(workspace, name="MyGeoImage")
+    geoimage: GeoImage = GeoImage.create(workspace, name="MyGeoImage")
```
[the remark this answers is not in the document]
Comment (sebhmg) [reply tests/geo_image_test.py:63]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (sebhmg) [reply tests/geo_image_test.py:63]: ho, this report seems correct... Does GeoImage.create() claim to return Entity | None ?
Should it instead declare return type GeoImage?
If possibly returning None, should assert here not None?

**Review by sebhmg — CHANGES_REQUESTED** [review]
please see note about type error reported by sonatype

File: tests/geo_image_test.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sebhmg) [reply tests/geo_image_test.py:63]: @sonatype-lift unignore

[the remark this answers is not in the document]
Comment (domfournier) [reply tests/geo_image_test.py]: Done

File: tests/geo_image_test.py [hunk 2]
```diff
@@ -155,7 +155,7 @@ def test_create_copy_geoimage(tmp_path):
     geoimage.vertices = geoimage.vertices
 
     # Test copy from extent that clips one corner
-    new_image = geoimage.copy(extent=[[9, 9], [10, 10]])
+    new_image = geoimage.copy_from_extent(np.vstack([[9, 9], [10, 10]]))
```
[the remark this answers is not in the document]
Comment (domfournier) [reply tests/geo_image_test.py:161]: Should have been addressed

**Comment by sebhmg** [comment]
pre-commit.ci run

**Review by sebhmg — APPROVED** [review]

**Pull request merged**